plugins/github-deployments/src/api/index.ts

```diff
@@ -26,12 +60,17 @@ export type GithubDeployment = {
   };
 };
 
+type QueryOptions = {
+  owner: string;
+  repo: string;
+  last: number;
+};
+
 export interface GithubDeploymentsApi {
-  listDeployments(options: {
-    owner: string;
-    repo: string;
-    last: number;
-  }): Promise<GithubDeployment[]>;
+  listDeployments(
+    options: QueryOptions,
+    location: string[],
```

Comment:
I'd move this into the `QueryOptions`, that or make the second parameter another options object, in case we need to extend it further.